Sources/BoolExtensions.swift

```diff
@@ -8,10 +8,11 @@
 
 extension Bool {
     /// EZSE: Converts Bool to Int.
-    public var toInt: Int { return Int(self) }
+    public var toInt: Int { return self ? 1 : 0 }
 
     /// EZSE: Toggle boolean value.
-    public mutating func toggle() -> Bool {
+    @discardableResult public mutating func toggle() -> Bool {
```

Comment:
It allows ignore returning value (no warning generated)

For example, I want to mutate Bool to its opposite, and not store the result, if not discardable I need to write:

`_ = b.toogle()` instead of `b.toogle()`